plugins/com.google.cloud.tools.eclipse.appengine.whitelist/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,19 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Cloud Tools for Eclipse App Engine JRE Whitelist Checks
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.whitelist;singleton:=true
+Bundle-Version: 94.0.0.qualifier
+Bundle-Vendor: GOOGLE
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Require-Bundle: org.eclipse.wst.server.core;bundle-version="1.7.0",
```

Comment:
Assuming this bundle is basically implementing a compilation participant, we don't need any of the JST and WST components, but will need JDT.